private/private.h

```diff
@@ -189,9 +189,11 @@ void _dispatch_prohibit_transition_to_multithreaded(bool prohibit);
 
 #if TARGET_OS_MAC
 typedef mach_port_t dispatch_runloop_handle_t;
-#elif defined(__linux__) || defined(__FreeBSD__)
+#elif defined(__linux__)
 typedef int dispatch_runloop_handle_t;
-#elif defined(__unix__) && !defined(__linux__) && !defined(__FreeBSD__)
+#elif defined(__FreeBSD__)
+typedef uint64_t dispatch_runloop_handle_t;
```

Comment:
Do we need this case? The case below looks like it's is typedef'ing the same things and the `defined(__unix__)` should capture on FreeBSD as well.